### PR TITLE
Add VPN state to broken sites report

### PR DESCRIPTION
**Pull request by aitorvs**
Task/Issue URL: https://app.asana.com/0/488551667048375/1207135101272466/f

### Description
Send `vpnOn` parameter to broken site reports

### Steps to test this PR
- [x] install this build and launch app
- [x] enable VPN
- [x] load a page and send broken site report (logcat filter by `vpnOn`
- [x] verify that `epbf` pixel is seng and contains `vpnOn` param to `true`
- [ ] disable VPN and send broken site report
- [ ] verify that `epbf` pixel is seng and contains `vpnOn` param to `false`

### CODE REVIEW

**Comment by aitorvs** [comment]
* **#4448** <a href="https://app.graphite.dev/github/pr/duckduckgo/Android/4448?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `develop`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @aitorvs and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: app/src/main/java/com/duckduckgo/app/brokensite/api/BrokenSiteSender.kt [hunk 1]
```diff
@@ -83,6 +85,8 @@ class BrokenSiteSubmitter @Inject constructor(
                 featureToggle.isFeatureEnabled(PrivacyFeatureName.ContentBlockingFeatureName.value) &&
                 !contentBlocking.isAnException(brokenSite.siteUrl)
 
+            val vpnOn = runCatching { networkProtectionState.isRunning() }.getOrNull()
```
Comment (karlenDimla) [inline app/src/main/java/com/duckduckgo/app/brokensite/api/BrokenSiteSender.kt:88]: why not `.getOrDefault(false)`?

Comment (aitorvs) [reply app/src/main/java/com/duckduckgo/app/brokensite/api/BrokenSiteSender.kt:88]: because `isRunning` failing doesn't mean VPN is disabled.

Comment (karlenDimla) [reply app/src/main/java/com/duckduckgo/app/brokensite/api/BrokenSiteSender.kt:88]: Right. We want to differentiate that ( by setting null).

**Review by karlenDimla — APPROVED** [review]

**Pull request merged**